PartitioningAgent/Agent.cs

```diff
@@ -37,16 +42,20 @@ public class Agent : IPartitioningAgent
             IThreadWrapper thread,
             IClusteringConfig clusteringConfig,
             IFactory factory,
-            ILogger logger)
+            ILogger logger,
+            IAzureManagementAdapterClient azureManagementAdapter)
         {
             this.clusterNodes = clusterNodes;
             this.partitions = partitions;
             this.simulations = simulations;
             this.thread = thread;
             this.factory = factory;
             this.log = logger;
+            this.azureManagementAdapter = azureManagementAdapter;
+            this.clusteringConfig = clusteringConfig;
             this.checkIntervalMsecs = clusteringConfig.CheckIntervalMsecs;
             this.running = false;
+            this.currentNodeCount = 1;
```

Comment:
Create a constant DEFAULT_NODE_COUNT and use that here and in line 118

Reply:
updated